models/_sector/nft/trades/ethereum/platforms/archipelago_ethereum_base_trades.sql

```diff
@@ -0,0 +1,78 @@
+{{ config(
+    schema = 'archipelago_ethereum',
+    alias ='base_trades',
```

Comment:
do you have a strong opinion on using `base_` on table name and file name? is it to overly express that this is base layer or can we remove and just assume it's base layer?

Reply:
since i'm already commenting a lot, assume all comments in `archipelago` apply to other platform base trades too

Reply:
No strong opinion on `base_` but I do think having both `archipelago_ethereum_trades` and `nft_ethereum_trades` when they don't follow the same schema is confusing. Hence the introduction of `base_trades`.
Very open to other naming suggestions.

Reply:
i think i'm good with `base_` prefix. not sure many users are using these base tables anyway, ultimately just using `nft.trades`